CSparse/CSparse.csproj

```diff
@@ -25,6 +25,10 @@
 </PackageReleaseNotes>
   </PropertyGroup>
 
+  <ItemGroup>

```

Comment:
As soon as this gets fixed (by adding a conditional check for netstandard2.0, see comment in issue #41 ) the pull request should be ready for merge.

Reply:
Thanks for sharing your changes, I have completed specifying the target framework to Sytem.Memory.